roxie/udplib/udptrs.cpp

```diff
@@ -774,7 +774,8 @@ class UdpReceiverEntry : public IUdpReceiverEntry
             }
             output_queue = new queue_t[numQueues];
             maxPktsPerQ = new int[numQueues];
-            for (unsigned j = 0; j < numQueues; j++) 
+            maxPktsPerQ[0] = INT_MAX;   // Always send out-of-band first
```

Comment:
Not sure it needs to be higher. There's less traffic on it than on the main queues, I would think (though the ACKs will certainly increase that amount) and it will clear much more quickly.

I have pushed a fix. I can increase the limit too if you think advisable, though arguably that would be a separate bug fix...